platforms/java/com/area9innovation/flow/Database.java

```diff
@@ -124,21 +124,17 @@ public RSObject(DBObject dbo, String query, Object[] queryParams) throws SQLExce
         }
     }
 
-	public void initializeDatabase() {
+	public void initialize() {
```

Comment:
We need `initializeDatabase` there https://github.com/area9innovation/innovation/blob/e7b73900bed09f1911f85031bf3c5dd49779cf62/platforms/java/com/area9innovation/flow/DatabaseSValue.java#L13